src/2981R0_improving_our_safety_with_a_physical_quantities_and_units_library.md

```diff
@@ -1024,7 +1029,15 @@ static_assert(5 * h / (120 * min) == 0 * one);
 
 This is why floating point representation types are recommended as a default to store the numerical
 value of a quantity. Some popular physical units libraries even
-[forbid integer division at all](https://aurora-opensource.github.io/au/main/troubleshooting/#integer-division-forbidden).
+[forbid integer division at all](https://aurora-opensource.github.io/au/main/troubleshooting/#integer-division-forbidden). 
+
+The problem is similar to the one described in the section about accidential truncation of values 
+through conversion. While the consequent use of floating point representation types may be a good idea,
+it is not always possible. Especially in close-to-the-metal applications and small embedded sytem the 
+use floating point types is sometimes not an option, either for performance reason or lack of hardware support.
+Other than with cast operations offering an explicitely marked unsafe operator mathematical operator is more 
+diffcult as it would force users to use different symbols for quantities using floating point or integral values. 
+As such the need to explicitely enabling such unsafe operations through a compiler flag could be an option. 
```

Comment:
I already left a comment saying that floating-point is a term with a dash: <https://github.com/mpusz/wg21-papers/commit/50e7482228b5396119911aa0977cf24aeccc01ab#r129455251>.

```suggestion
The problem is similar to the one described in the section about accidental truncation of values 
through conversion. While the consequent use of floating-point representation types may be a good idea,
it is not always possible. Especially in close-to-the-metal applications and small embedded systems, the 
use floating-point types is sometimes not an option, either for performance reasons or lack of hardware support.
Other than with cast operations offering an explicitly marked unsafe operator mathematical operator is more 
difficult as it would force users to use different symbols for quantities using floating-point or integral values. 
As such, the need of explicitly enabling such unsafe operations through a compiler flag could be an option. 
```